src/scripts/Add-Scope.ps1

```diff
@@ -0,0 +1,241 @@
+param(
+  [string]$hubName,
+  [string]$Scope,
+  [string]$StorageAccountId,
+  [int]$numberOfMonths = 2,
+  [string]$ContainerName = "ms-cm-exports",
+  [string]$FolderName = $Scope,
+  [string]$Metric = "amortizedcost",
+  [int]$TimeOutMinutes = 15,
+  [int]$SleepInterval = 10
+)
+
+$ErrorActionPreference = "Stop"
+[bool]$Future = $true
+[bool]$History = $true
+if($numberOfMonths -lt 2) {$History = $false}
+[datetime]$today = Get-Date
+[datetime]$StartDate = $today.AddDays(-$today.Day + 1).AddMonths(-$numberOfMonths)
+[datetime]$EndDate = $today.AddDays(-$today.Day + 1).AddMonths(-1).AddDays(-1)
+
+function Write-DebugInfo {
+  param (
+    $DebugParams
+  )
+
+  Write-Host ("{0}    {1}    {2}" -f (Get-Date), $DebugParams.Name, $DebugParams.DefinitionTimeframe)
+}
+
+function Set-CostManagementApi {
```

Comment:
nit: This name isn't clear. Is there a more explicit name for what this is doing?